packages/core/addon/models/bard-request/request.js

```diff
@@ -82,6 +83,7 @@ export default Fragment.extend(Validations, {
   intervals: fragmentArray('bard-request/fragments/interval', {
     defaultValue: []
   }),
+  dataSource: DS.attr('string', { defaultValue: getDefaultDataSourceName() }),
```

Comment:
Would the saved request fragments without the `datasource` prop still work?

Reply:
Yes.